src/Components/ScrollToTopBtn.css

```diff
@@ -7,3 +7,11 @@
   border-radius: 56%;
   background: white;
 }
+
+@media screen and (hover: hover) {
+  .scrollToTop-btn:hover {
+    background: black;
+    border: 2px black solid;
+    transition: all 0.3s ease-in;
+  }
+}
```

Comment:
We are using PrimeReact, I think this can be achieve with its utility classes?
https://www.primefaces.org/primeflex/transitionproperty

Reply:
Do you need any help looking into this @RohitKS7 ?

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question